README.MD

```diff
@@ -1,18 +1,18 @@
-# DCE Tools
+# Export Tools
 
-This project provides Bazaarvoice Displayable Content Export tools written in different languages (Node.js, Python, etc.) to help developers experiment with the Displayable Content Export and to see what the response would be like. These example applications are meant as both a reference implementation of how to use the Bazaarvoice API, as well to get an idea of the format of the retrieved data.
+The Bazaarvoice Export tool was created to help developers experiment with Bazaarvoice export APIs ([DCE](https://developer.bazaarvoice.com/displayable-content-export),[PSI](https://developer.bazaarvoice.com/product-sentiment-insights-api/home)). The example applications are meant as both a reference implementation of how to use the Bazaarvoice export APIs, as well to get an idea on the content that is available. The sample is written in different languages (Node.js, Python, etc.). 
 
 To see what Displayable Content Export is, please refer to the Bazaarvoice [documentation](https://developer.bazaarvoice.com/displayable-content-export) for details.
 
+To see what Product Sentiment Insights API is, please refer to the Bazaarvoice [documentation](https://developer.bazaarvoice.com/product-sentiment-insights-api/home) for details.
+
 ## License
 This project is licensed under the Apache 2.0 License - see the [LICENSE](LICENSE) file for details.
 
 ## Reference Tools
-The Displayable Content Export is an HTTP-protocol based API, and as such, is not bound to any specific programming language. Your application might be written in Java, javascript, Ruby, Python, Perl, C#, C++, or any other programming language in which HTTP requests can be made over the internet.
-
-The reference languages contained in this project are simple examples of how one might interact with the Displayable Content Export. These examples are not meant for production use. Your implementation will require exception handling, HTTP retries in case of network failures, etc.
+Bazaarvoice export APIs are HTTP-protocol based API, and as such, is not bound to any specific programming language. Your application might be written in Java, javascript, Ruby, Python, Perl, C#, C++, or any other programming language in which HTTP requests can be made over the internet.
```

Comment:
"Bazaarvoice export APIs are HTTP-protocol based API, and as such, is not bound to any specific programming language."

Mixed plural and singular the that first part is a bit repetitive. How about "Bazaarvoice export APIs based on the HTTP-protocol, and as such, are not bound to any specific programming language."

Reply:
fixed with Frederick's suggestion